exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -98,6 +100,8 @@
  * String into ByteBuf.
  */
 public class DrillClient implements Closeable, ConnectionThrottle {
+  public static final String DEFAULT_CLIENT_NAME = "Apache Drill Java client";
```

Comment:
The C++ client will have a default Apache Drill C++ client, with an option to override it by the user